tests/tantivy_test.py

```diff
@@ -103,6 +103,26 @@ def test_query_errors(self, ram_index):
             index.parse_query("bod:men", ["title", "body"])
 
 
+class TestUpdateClass(object):
+
+    def test_delete_update(self, ram_index):
+        query = ram_index.parse_query("Frankenstein", ["title"])
+        top_docs = tantivy.TopDocs(10)
+        result = ram_index.searcher().search(query, top_docs)
+        assert len(result) == 1
+
+        writer = ram_index.writer()
+        writer.delete_documents('title', "frankenstein")
```

Comment:
*nitpick* maybe use the same quotes for fields ands the value to avoid confusing users.